torch/_inductor/lowering.py

```diff
@@ -2594,6 +2593,10 @@ def max_pool2d_with_indices(
         return fallback_max_pool2d_with_indices(
             x, kernel_size, stride, padding, dilation, ceil_mode
         )
+    if dilation != 1:
```

Comment:
you can combine this with **if window_size > 25**: **if window_size > 25 or (dilation != 1 or any(d != 1 for d in dilation):**? please also add a test case in https://github.com/pytorch/pytorch/blob/master/test/inductor/test_torchinductor.py#L2472

Reply:
Thank you! I've combined the if conditions. I'm now working on adding a test case.